data/masses/c_diff_monomer_masses.csv

```diff
@@ -1,4 +1,4 @@
-﻿Structure,Monoisotopicmass
+Structure,Monoisotopicmass
```

Comment:
What's changed here?

Reply:
No idea, couldn't see it myself either. :shrug: